core/src/test/scala/integration/kafka/api/IntegrationTestHarness.scala

```diff
@@ -42,8 +42,8 @@ trait IntegrationTestHarness extends KafkaServerTestHarness {
   lazy val consumerConfig = new Properties
   lazy val serverConfig = new Properties
 
-  val consumers = Buffer[KafkaConsumer[Array[Byte], Array[Byte]]]()
   val producers = Buffer[KafkaProducer[Array[Byte], Array[Byte]]]()
+  val consumers = Buffer[KafkaConsumer[Array[Byte], Array[Byte]]]()
```

Comment:
It is a side effect of making change and reverting change in the process of implementing this patch. I have changed it to the original order.